packages/cli/src/commands/generate/sdl/sdl.js

```diff
@@ -44,7 +44,16 @@ const missingIdConsoleMessage = () => {
   )
 }
 
-const modelFieldToSDL = (field, required = true, types = {}) => {
+const addFieldGraphQLComment = (field, str) => {
+  const description = field.documentation || `Description for ${field.name}.`
+
+  return `"""
+  ${description}
+  """
```

Comment:
Can we be more clever here and do `"""${description}"""` if `description` doesn't contain any newlines? (And maybe also check the character count, so it's not too long)

Reply:
Yup. fixed!